src/Workspaces/Remote/ServiceHub/Host/TestUtils.cs

```diff
@@ -184,32 +184,34 @@ public static Task AppendAssetMapAsync(this Solution solution, Dictionary<Checks
         public static async Task AppendAssetMapAsync(
             this Solution solution, Dictionary<Checksum, object> map, ProjectId? projectId, CancellationToken cancellationToken)
         {
+            var callback = (Checksum checksum, object asset) => { map[checksum] = asset; };
```

Comment:
This is test code.  So this just maps from the new api to the dictionary-based approach the test uses to validate results.